app/controllers/application_controller.rb

```diff
@@ -1,8 +1,23 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  include Pundit
```

Comment:
DEPRECATION WARNING: 'include Pundit' is deprecated. Please use 'include Pundit::Authorization' instead.